website/static/js/pages/project-registrations-page.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+require('css/registrations.css');
+
+var ko = require('knockout');
+var $ = require('jquery');
+
+var $osf = require('js/osfHelpers');
+var RegistrationManager = require('js/registrationUtils').RegistrationManager;
+
+var ctx = window.contextVars;
+var node = window.contextVars.node;
+
+$(document).ready(function() {
+
+    $('#registrationsTabs').tab();
+    $('#registrationsTabs a').click(function (e) {
+        e.preventDefault();
+        $(this).tab('show');
+    });
+
+    var draftManager = new RegistrationManager(node, '#draftRegistrationScope', {
```

Comment:
It seems that this item was actively underway when this was moved to code review. Can we get a final decision from you two on whether this needs to be resolved (as part of the action items) before moving forward?